taskset.go

```diff
@@ -312,6 +315,11 @@ func (ts *TaskSet) SavePendingChanges() {
 			task.SaveToDisk()
 		}
 
+		if task.Deleted {
+			relativePath := filepath.Join(task.Status, task.UUID+".yml")
```

Comment:
I think the task struct needs a `Delete()` method to be consistent with the path resolution and abstraction afforded by `SaveToDisk()` which does `filepath := MustGetRepoPath(t.Status, t.UUID+".yml")`

Note that giving git an absolute filepath the delete is fine.

Reply:
On second thoughts, I consider deletion to be part of the job of `SaveToDisk()` as that already has a mechanism to delete all matching tasks in all directories (important if there are manual changes to the repository which could result in duplicate tasks.